content/blog/2016/2016-03-11-new-jenkins-20-preview.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "New Jenkins 2.0 Preview Build"
+tags:
+- jenkins2
+author: daniel-beck
+---
+
+We just published the new Jenkins 2.0 preview build alpha 3.
+
+## What's new?
+
+* **Jenkins is now secure out of the box**:
+  Administrators previous had to set up authentication and authorization while Jenkins was accessible to anyone on the same network.
+  Now, Jenkins is protected out of the box, so that it is always safe from unauthorized access.
+* **Plugin selection for setup**:
+  We refined the plugin selection on the setup dialog.
+  You've always wondered why Jenkins does not install the Git Plugin by default?
+  Now it does.
```

Comment:
Perhaps: "Now it does, along with a number of other plugins popular in the Jenkins community."